scripts/Map.cs

```diff
@@ -0,0 +1,62 @@
+using System;
+
+using Game.Scripts;
+
+using Godot;
+
+
+public partial class Map : Node2D
+{
+    private Map _map;
+    private Core core; // Deklaration des Core-Objekts
+    private Player _player;
+    private Ally _ally;
+    private Health _allyHealth;
+    static double _timeElapsed = 0f;
+    private float _allyHealthInterval = 3f;
+
+    public override void _Ready()
+    {
+        _map = this;
+        core = GetNode<Core>("%Core"); // Initialisierung in _Ready
+        GD.Print(core.Scale); // Ausgabe der Skalierung in der Godot-Konsole
+        _player = GetNode<Player>("%Player");
+        _ally = GetNode<Ally>("%Ally");
```

Comment:
%Ally shouldnt be used, you can only use the percent sign to get a node, if you can be sure, there is only one Node of that name in the Game. Like for Player or Core. But since we want to have multiple Allys, this should be changed